src/uu/install/src/install.rs

```diff
@@ -426,18 +426,25 @@ fn copy_files_into_dir(files: &[PathBuf], target_dir: &PathBuf, b: &Behavior) ->
 
     let mut all_successful = true;
     for sourcepath in files.iter() {
-        let targetpath = match sourcepath.as_os_str().to_str() {
-            Some(name) => target_dir.join(name),
-            None => {
-                show_error!(
-                    "cannot stat '{}': No such file or directory",
-                    sourcepath.display()
-                );
+        if !sourcepath.exists() {
+            show_error!(
+                "cannot stat '{}': No such file or directory",
```

Comment:
is it the same output as in GNU?

Reply:
Yes

```
┌SLATEデnomiusガ14:58:57時install_bugfix=デ「~/P/c/bugtest」
└$ tree
.
├── Lib
│   └── unittest
│       ├── asd
│       │   └── stuff
│       └── runner.py
└── lib
    └── unittest
        └── runner.py

5 directories, 3 files
┌SLATEデnomiusガ14:59:00時install_bugfix=デ「~/P/c/bugtest」
└$ install ./Lib/unittest/{runner.py,asd} lib/unittest
install: omitting directory './Lib/unittest/asd'
```
(install is system install, not uutils)

Reply:
oh, I answered a different question.
This particular output is the same as before. although it should be `show_info!()` not `show_error!()`

Reply:
fixed